sdk/textanalytics/ai-text-analytics/src/analyzeSentimentResult.ts

```diff
@@ -68,3 +171,84 @@ export function makeAnalyzeSentimentErrorResult(
 ): AnalyzeSentimentErrorResult {
   return makeTextAnalyticsErrorResult(id, error);
 }
+
+/**
+ * Converts a sentence sentiment object returned by the service to another that 
+ * is user-friendly.
+ *
+ * @param sentence - The sentence sentiment object to be converted.
+ * @param response - The entire response returned by the service.
+ * @returns The user-friendly sentence sentiment object.
+ */
+function convertGeneratedSentenceSentiment(
+  sentence: GeneratedSentenceSentiment,
+  response: GeneratedClientSentimentResponse
+): SentenceSentiment {
+  return {
+    confidenceScores: sentence.confidenceScores,
+    sentiment: sentence.sentiment,
+    text: sentence.text,
+    minedOpinions: sentence.aspects?.map(
+      (aspect: SentenceAspect): MinedOpinion => ({
+        aspect: {
+          confidenceScores: {
+            ...(aspect.confidenceScores as SentimentConfidenceScores),
+            neutral: 0
+          },
```

Comment:
@willmtemple is this the right way to handle the comment here: https://github.com/Azure/azure-sdk-for-js/pull/10484#discussion_r467214326?

Reply:
I'm on the fence about this. Seems weird to have a "fake" property in the response when it is always zero, especially given that the service spec provides a separate type for this, but if we decided to do this pre-emptively I won't lose sleep about it.

Maybe @maririos or @iscai-msft can add some texture to why this is set to zero.

Reply:
We decided to have `neutral = 0` so we can reuse the type `SentimentConfidenceScores` that we use in `SentenceSentiment` and `DocumentSentiment`

Reply:
@willmtemple, do we have guidelines in TS around a type can be unnecessarily big to avoid creating another smaller one?